pkg/backup/item_backupper.go

```diff
@@ -433,12 +449,33 @@ func (ib *itemBackupper) executeActions(
 			if err != nil {
 				return nil, itemFiles, errors.WithStack(err)
 			}
-
-			_, additionalItemFiles, err := ib.backupItem(log, item, gvr.GroupResource(), gvr, mustInclude, finalize)
-			if err != nil {
-				return nil, itemFiles, err
+			if additionalItem.GroupResource == kuberesource.PersistentVolumeClaims {
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					_, additionalItemFiles, err := ib.backupItem(log, item, gvr.GroupResource(), gvr, mustInclude, finalize)
+					if err != nil {
+						errChannel <- err
+						return
+					}
+					additionalItemFilesChannel <- additionalItemFiles
+				}()
+			} else {
+				_, additionalItemFiles, err := ib.backupItem(log, item, gvr.GroupResource(), gvr, mustInclude, finalize)
+				if err != nil {
+					return nil, itemFiles, err
+				}
+				itemFiles = append(itemFiles, additionalItemFiles...)
 			}
-			itemFiles = append(itemFiles, additionalItemFiles...)
+		}
+		wg.Wait()
+		close(additionalItemFilesChannel)
```

Comment:
I'll refine the channel stuff / code accuracy later, currently consider this only for representing draft idea on approach.